packages/openjdk-21/build.sh

```diff
@@ -0,0 +1,102 @@
+TERMUX_PKG_HOMEPAGE=https://openjdk.java.net
+TERMUX_PKG_DESCRIPTION="Java development kit and runtime"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=21.0.3+6
```

Comment:
Why 21.0.3+6 and not latest 21.0.4+6 (or 21.0.3+9)?

Reply:
Random number is just for testing though. Not using adapted 21.0.1 just like other distribution like androidide, pojav is for testing the patches on future update versions, while not using the latest is ~~idk which version is newest~~ (get the version rule at openjdk wiki) for testing termux auto update.

Reply:
What do you mean, what is random with the version string? 21.0.4+6 is latest, git log jdk-21.0.3-ga..jdk-21.0.4+6 shows that it has 302 additional commits.

Reply:
> What do you mean, what is random with the version string? 21.0.4+6 is latest, git log jdk-21.0.3-ga..jdk-21.0.4+6 shows that it has 302 additional commits.

when creating the package I have not tested the auto update, concerning that patch update with lower update version would make update script think that this need upgrade. After doing some testing to the function I figure out that it's not a concern though, but I still want to fix suffix with `-ga` to:
- lower maintain burden while provides regular updates
- use something like "stable" version, as it's the final patch version. (https://wiki.openjdk.org/display/JDKUpdates/JDK+21u#Timelines)